src/EventListener/FileUploadListener.php

```
@@ -52,6 +54,10 @@ public function onFileUpload(FileUploadEvent $event): void
             return;
         }
 
+        if (Validator::isUuid($filePath)) {
+            $filePath = FilesModel::findByUuid($filePath)->path;
```

This could still return `null` under some weird circumstances. In such case we should either throw an exception or handle it the same way as above, i.e. set a correct response.

---

Done :)